README.md

```diff
@@ -21,18 +21,44 @@ For a more in-depth overview of openCryptoki, please refer to manual
 Does not work with TPM version 2.0. We plan to remove the TPM token in a future 
 openCryptoki release or version.
 
+AIX only supports the CCA and software tokens, and both are enabled in a
+default build configuration. Other tokens are unsupported and cannot be
+force-enabled, even through the `configure` utility.
+
 ## REQUIREMENTS:
 
+### Common
+Building opencryptoki needs the following utilities.
+- flex
+- bison
+- make
+- autoconf
+- automake
```

Comment:
I guess `libtool` should also be mentioned, at least foe Linux.

Reply:
`libtool` is already mentioned - see two lines below. It is also used on AIX, so there's no need to mark the utility platform-specific.

Reply:
ah, sure, my fault, should have seen it :-)